src/features/makeOtc/makeOtcActions.ts

```diff
@@ -92,6 +94,9 @@ export const createOtcOrder = createAsyncThunk(
       };
 
       dispatch(setUserOrder(fullOrder));
+      if (params.activeIndexers) {
```

Comment:
Only send to indexer if order is public

```suggestion
      if (params.activeIndexers && fullOrder.senderWallet === nativeCurrencyAddress) {
```